readthedocs/doc_builder/python_environments.py

```diff
@@ -89,14 +89,17 @@ def install_package(self, install):
                 bin_path=self.venv_bin(),
             )
 
-    def venv_bin(self, filename=None):
+    def venv_bin(self, filename=None, conda=False):
         """
         Return path to the virtualenv bin path, or a specific binary.
 
         :param filename: If specified, add this filename to the path return
         :returns: Path to virtualenv bin or filename in virtualenv bin
         """
-        parts = [self.venv_path(), 'bin']
+        parts = ["$READTHEDOCS_VIRTUALENV_PATH", "bin"]
+        if conda:
```

Comment:
This seems like an awkward place to introduce conda. Previously this function didn't know about conda at all, and I'm wondering if it's the right place for this logic? Or perhaps we need to rename this function?

Reply:
I updated this to make the code more explicit: 66fe0a7d9. Let me know if that looks better 👍🏼

Reply:
That is definitely nicer. Though I'd argue the `venv` in the name is misleading for conda envs :) Perhaps it should just be called `env_bin` or similar? Not a huge deal tho.